src/main/java/org/apache/sysds/hops/BinaryOp.java

```diff
@@ -496,7 +498,7 @@ else if (mbin == MMBinaryMethod.MR_BINARY_M) {
 				setOutputDimensions(binary);
 				setLineNumbers(binary);
 				setLops(binary);
-			}
+			} else throw new HopsException("Lop construction not implemented for ExecType " + et);
```

Comment:
split line

Reply:
Why? It is not exceeding the line character limit.

Reply:
in this pr you do it two different ways, i liked the first better with newline and brackets. I personally would prefer if there was a newline for the throw, if you want to include brackets or not is up to you.

Reply:
I have now changed the other throw so that it is also on the same line. 
I generally think that it is acceptable to have both, since it makes more sense in some cases to have it on a single line. It depends on the context. If, for instance, the other if-else parts are without brackets then it makes more sense to have the throw on a single line without brackets as well.